src/components/PresenceMap/component.js

```diff
@@ -0,0 +1,141 @@
+import React, { useRef, useState, useEffect } from 'react';
```

Comment:
the name of the file should be index.js, that way the could import from `'src/component/PresenceMap';`

Reply:
There is an **index.js** already that imports this **component.js**. PresenceMap is imported in `src/components/index.js` and you can test it in http://localhost:6060/#/PresenceMap. Is that what you mean?